util.c

```diff
@@ -2135,6 +2135,16 @@ any of the categories are by default enabled.
 =for apidoc vwarner
 This is like C<L</warner>>, but C<args> are an encapsulated argument list.
 
+=for apidoc fatal_warner
+
+This outputs a warning as if fatal warnings were enabled.  This may
+return in some cases.
```

Comment:
I can't quite follow from these words what it is that the function does. Can you perhaps use more words to describe it?